phantom_connector.py

```diff
@@ -21,19 +21,21 @@
 import gzip
 import json
 import os
+import pathlib
 import random
 import socket
 import string
 import tarfile
 import time
 import zipfile
 from pathlib import Path
+from typing import Tuple
 
-import magic
```

Comment:
Looks like that `magic` dir confuses the tool...

Reply:
Should not. Magic is a dir, but not a package, nor module. But you're right, the import optimization tool has moved it anyways 🫨 Not a big issue here, but perhaps I will rename `magic` to `magic_files` then.

Reply:
It's funny because pycharm imports optimization does that right: putting magic and requests along with the local packages, so not confusing anything. But isort is confused and does not put it to the separate group of dependencies.

Reply:
Yeah, a basic check for something (not) being a package is not that hard and pretty important for a tool like that :D